src/annotator/test/guest-test.coffee

```diff
@@ -4,18 +4,13 @@ Plugin = require('../plugin')
 
 Delegator = require('../delegator')
 $ = require('jquery')
-Delegator['@noCallThru'] = true
```

Comment:
This `@noCallThru` property doesn't do anything any more. It is left over from when we used proxyquire. I'm just removing it here as a piece of clean-up.